app/src/main/res/xml/quran_preferences.xml

```diff
@@ -147,29 +147,16 @@
   </PreferenceCategory>
 
   <PreferenceCategory
```

Comment:
in an ideal world, the patch would have been as simple as Mohamed's original patch where he'd just add `PreferenceScreen` around it - unfortunately though it doesn't work. maybe it would work with support-v7 fragments, but we can try that later?